index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css">
+    <title>Tech Start Up Conference</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+	<header class="header-container">
+		<nav class="header-nav">
+				<div class="container-top-nav">
+						<div class="header-top-nav">
+								<ul class="header-top-nav-desktop">
+										<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+										<li><a href="#"><i class="fab fa-twitter"></i></a></li>
+										<li><a href="#">English</a></li>
+										<li><a href="#">My Page</a></li>
+										<li><a href="#">Logout</a></li>
+								</ul>
+						</div>
+				</div>
+				<div class="container-nav">
+						<div class="logo-nav">
+								<a href="./index.html">
+										<img src="./assets/logo_ccglobal.png" alt="logo">
+								</a>
+						</div>
+						<ul class="header-nav-desktop">
+								<li><a href="./about.html">About</a></li>
+								<li><a href="#">Program</a></li>
+								<li><a href="#">Join</a></li>
+								<li><a href="#">Sponsor</a></li>
+								<li><a href="#">News</a></li>
+								<li><a href="#">CC Campaign</a></li>
+						</ul>
+				</div>
+		</nav>
```

Comment:
Please ensure that the pages should look almost identical to the original design. I understand that you use `border box` as the `hover effect` for your `nav link` but in the original design, the last `item` has a `border red box` around it. Kindly implement that according to the project requirement. 

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163294797-d4e54ccc-5b88-4420-ad14-d4375cf2b177.png) | ![image](https://user-images.githubusercontent.com/10332499/163294751-9abfecf5-5c8d-4952-8526-825f891e0ae4.png)